src/plugins/intel_cpu/src/transformations/cpu_opset/convert_to_cpu_specific_opset.hpp

```diff
@@ -31,7 +31,40 @@ inline void ConvertToCPUSpecificOpset(std::shared_ptr<ov::Model> &model) {
 
     ov::pass::Manager manager("CPU:ConvertToCPUSpecificOpset");
     manager.set_per_pass_validation(false);
+
+    // CPU_REGISTER_PASS_COMMON(manager, AlignMatMulInputRanks);
     CPU_REGISTER_PASS_COMMON(manager, ConvertMatMulToFC);
+    if (std::getenv("EXTRA_DUMP")) {
+        manager.run_passes(model);
+        ov::pass::Serialize("after_fc.xml", "/dev/null").run_on_model(model);
+        CPU_DISABLE_PASS_COMMON(manager, ConvertMatMulToFC);
+    }
+
+    std::vector<ov::element::Type> supported_compression_types {
+        ov::element::u8,
+        ov::element::i8,
+        ov::element::u4,
+        ov::element::i4,
+        ov::element::nf4,
+        ov::element::f4e2m1,
+    };
+
+    CPU_REGISTER_PASS_X64(manager, pass::ConvertFullyConnectedToFullyConnectedCompressed,
+                          supported_compression_types,
```

Comment:
1. Original graph optimizer pass has limitation on activations precision. Is it handled inside the transformation? 
2. I also don't see limitation on avx2 precision. 
3. https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/graph_optimizer.cpp#L436-L449 seems to be missed.
4. We use place HW specific limitations inside isSupportedOperation function. Worth to align the behavior

Reply:
Correct.
I am not sure about moving the check into `isSupported`
The fact that reference implementation is going to be executed should not mean the operation is not supported. From my understanding `isSupported` is more about "not possible to execute at all"

Reply:
There is not reference impl inside oneDNN fork for compressed IP. So this is exaclty "not possible to execute at all" case

Reply:
Added isSupportedCompressed